packages/docmanager-extension/src/index.tsx

```diff
@@ -738,9 +738,19 @@ function addCommands(
     }
   });
 
+  const caption = () => {
+    if (isWritable()) {
+      return trans.__('Save and create checkpoint');
+    } else {
+      return trans.__(
+        'In collaborative mode, the document is saved automatically after every change'
```

Comment:
What about the case where the document is not writable and we are not in collaborative mode?

Reply:
Good point. I have pushed a commit to show the tooltip only in collaborative mode.